streams/src/test/java/org/apache/kafka/streams/integration/RegexSourceIntegrationTest.java

```diff
@@ -352,7 +352,9 @@ public StreamTask createStreamTask(TaskId id, Collection<TopicPartition> partiti
 
 
         void waitUntilTasksUpdated() {
```

Comment:
I'm assuming we are refactoring it in another JIRA / PR to make it more generic, for example taking a timeout parameter like Scala version: https://github.com/apache/kafka/blob/trunk/core/src/test/scala/unit/kafka/utils/TestUtils.scala#L765

We can leave it as is for now.